### DISCUSSION TITLE
`tabindex` on `p-button`

### DISCUSSION THREAD

**Discussion by joeferner** [primefaces/community#1058] [PrimeNG] [answered]
It doesn't look like there is a way to specify `tabindex` on a `p-button`. Is there some work around that I'm missing?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We're currently working on passthrough feature implementation. It'll allow users to pass any attribute or properties to the related component. We didn't add tabindex to the button, you can read the reason below from the MDN sources.
Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex


Some focusable HTML elements have a default tabindex value of 0 set under the hood by the user agent. 
These elements are an <a> or <area> with href attribute, <button>, <frame>, <iframe>, <input>, <object>, <select>, <textarea>, and SVG <a> element, or a <summary> element that provides summary for a <details> element. Developers shouldn't add the tabindex attribute to these elements unless it changes the default behavior (for example, including a negative value will remove the element from the focus navigation order).
